### PR TITLE
general po updates

### PR DESCRIPTION
**Pull request by dmedri**
po updates on available languages and minor fixes on it.po

### CODE REVIEW

File: po/da.po [hunk 1]
```diff
@@ -6,8 +6,8 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: systemd master\n"
-"Report-Msgid-Bugs-To: https://github.com/systemd/systemd/issues\n"
-"POT-Creation-Date: 2015-10-07 19:30+0000\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-07-15 13:11+0200\n"
```
Comment (keszybz) [inline po/da.po:10]: Why remove the url?

**Comment by keszybz** [comment]
Many of the formatting changes seem spurious, but they're not harmful, so I guess it's fine to pull them in if it'd be a lot of extra work to avoid them. As for the it translation changes, they seem OK, but it'd be better if somebody who speaks the language could vet them.

**Comment by poettering** [comment]
well, i'd probably just merge localized message patches as we cannot realistically review them, given that we don't speak all languages of this world. Hence, unless somebody objects I'd just merge this, but I do wonder about the URL that got lost there. @dmedri can you comment on that?

**Comment by keszybz** [comment]
>

**Comment by dmedri** [comment]
The "spurious" changes in the first patch are generated with "cd systemd/ && ./autogen.sh && ./configure && cd po/ && make update-po". Modified files are those with fuzzy, untranslated strings and changed references (gettext feature). About the URL, it make sense to pass the value as an argument in the previous sequence of commands to avoid empty fields.

**Comment by keszybz** [comment]
Most of the changes are inconsequential line wrapping changes. Changing the line length back and forth like this obscures the actual changes. I wonder if it would be possible to tweak the  settings that `make update-po` uses to make things consistent.

Similarly for the URL: can you add it as an argument?

**Comment by dmedri** [comment]
Reworking...

**Pull request closed without merging**